Cargo.toml

```diff
@@ -9,22 +9,21 @@ readme = "README.md"
 edition = "2018"
 
 [dependencies]
-atat = { git = "https://github.com/BlackbirdHQ/atat", branch = "master" }
-embedded-hal = "0.2"
-heapless = "0.5"
+#atat = { git = "https://github.com/BlackbirdHQ/atat", branch = "master" }
+# atat = { path = "../atat/atat" }
```

Comment:
Can you remove this line? (Feel free to leave in the commented-out git source on line 12 if you think that'll become useful.)